### ISSUE TITLE
Error parsing Lottie JSON with precomposition

### ISSUE THREAD

**Issue by seabassjh** [linebender/velato#4]
bodymovin-rs is unable to parse any animations with precompositions.

 Tested using several animations from https://googlefonts.github.io/noto-emoji-animation/.  One animation with a precomp looks like this: (using https://lottiefiles.github.io/lottie-docs/playground/json_editor/):
 
<img width="744" alt="image" src="https://github.com/linebender/velato/assets/20546772/3e0cfd0e-07b2-4a52-a3e1-2e5996a4166a">

### Error: ###
```
Unable to parse lottie JSON: ParseFailed(Error("data did not match any variant of untagged enum Asset", line: 1, column: 80831))
```

I believe it is referring to [this](https://github.com/BrainiumLLC/bodymovin-rs/blob/23e30f3aa8a1b935e9910dee90f8e218b1c8659c/src/sources/mod.rs#L10)  data structure not able to be deserialized
```
#[derive(Debug, Deserialize)]
#[serde(untagged)]
pub enum Asset {
    Image(Image),
    PreComp(PreComp),
}
```

**Comment by simbleau** [comment]
This issue can be closed as it will be solved via #13

**Comment by DJMcNab** [comment]
We should add this as a linked issue in #13, rather than pre-emptively closing it

I don't expect #13 to take much longer, but it's best to keep the correct state of the issue on `main` tracked.

I've added it as a linked issue.
